build.sbt

```diff
@@ -63,9 +63,18 @@ lazy val keeper = project
     fork in test := true
   )
 
+lazy val dockerSettings = Seq(
+  dockerBaseImage := "openjdk:11",
```

Comment:
We can probably go with something thinner here, like one of those alpine jre images.

Reply:
sure